src/components/inviteWelcome/InviteWelcome.js

```diff
@@ -0,0 +1,206 @@
+import React, { useCallback, useState } from 'react'
+import { View } from 'react-native'
+
+// import logger from '../../lib/logger/pino-logger'
+import { fireEvent, INVITEWELCOME_NEXT, INVITEWELCOME_SKIPPED } from '../../lib/analytics/analytics'
+import { withStyles } from '../../lib/styles'
+import { getShadowStyles } from '../../lib/utils/getStyles'
+import CustomButton from '../common/buttons/CustomButton'
+import Wrapper from '../common/layout/Wrapper'
+import Text from '../common/view/Text'
+import WalletSVG from '../../assets/Invite/wallet.svg'
+import SelfieSVG from '../../assets/Invite/selfie.svg'
+import MobileSVG from '../../assets/Invite/mobile.svg'
+
+import Section from '../common/layout/Section'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import { isShortDevice } from '../../lib/utils/mobileSizeDetect'
+import NavBar from '../appNavigation/NavBar'
+import { theme } from '../theme/styles'
+
+// const log = logger.child({ from: 'Welcome' })
+
+const steps = {
```

Comment:
1. better to add some constants e.g. STEP_INTO = 1, STEP_HOWTO = 2 etc
2. steps could be defined as an array with first empty element e.g. 
```javascript
const steps = [,
  {
    title: '// ...'
    // ...
  },
  // ...
```

Reply:
overkill

Reply:
@sirpy please try to look on that from the other side. You haven't worked on that code some time then you need to fix some bug. You open the code (you have forgot) and you see the check `if (currentStep === 2)`. What's "2" ? Is it correct ? As a result - a lot of time to remain it. In opposite if you'll se if (STEP_HOWTO === currentStep) you immediately understand everything

Reply:
you are right but still you need to look at the bigger context
this is a plain static wizard. ask yourself
1. is there anything special about each step?
2. are we likely to change the order?
3. is there complicated logic between steps?

i have no need to know this is the HOWTO step, it will anyways not  tell me much as there's no special fuctionality for each screen.
so in general you might be right but not in this case.
again I stress that you need to have critical thinking where refactoring have a big impact and where it doesnt.